settings.gradle

```diff
@@ -19,5 +19,6 @@ include ':examples',
         ':libnavui-resources',
         ':libnavui-voice',
         ':libnavui-speedlimit',
+        ':libnavui-lifecycle',
```

Comment:
```suggestion
        ':libnavigation-lifecycle',
```
This is more of a core module, isn't it?

Reply:
yeah i was going back and forth here 🤔 .. from comments above, am even considering moving to the same place as `MapboxNavigationProvider`

Reply:
Moved it